.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,40 @@
+name: Compile on Ubuntu
+
+on:
+  push:
+    paths-ignore:
+    - '**.md'
+    branches-ignore: [snapshot]
+  pull_request:
+    paths-ignore:
+    - '**.md'
+
+jobs:
+  linux:
+
+    runs-on: ubuntu-24.04
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        submodules: 'recursive'
+
+    - name: Install Qt
+      uses: jurplel/install-qt-action@v4
+      with:
+        version: 6.8.1
+        arch: linux_gcc_64
+        modules: ''
+        tools: 'tools_qtcreator'
```

Comment:
We don't need to install Qt Creator, that's only done in the Tiled workflow to get Qbs. Actually I'd prefer to compile against the system Qt anyway, so we can drop this step.